ggml/src/ggml-cuda/cross-entropy-loss.cu

```diff
@@ -126,7 +126,7 @@ void ggml_cuda_cross_entropy_loss(ggml_backend_cuda_context & ctx, ggml_tensor *
 #if !(defined(GGML_USE_HIP) && defined(__HIP_PLATFORM_AMD__))
         static bool shared_memory_limit_raised[GGML_CUDA_MAX_DEVICES] = {false};
         if (!shared_memory_limit_raised[id]) {
-            CUDA_CHECK(cudaFuncSetAttribute(cross_entropy_loss_back_f32<true>, cudaFuncAttributeMaxDynamicSharedMemorySize, smpbo));
```

Comment:
Hi @JohannesGaessler , @slaren ,
per my understanding, here it should be `cross_entropy_loss_f32`, please kindly confirm. Thanks.

Reply:
From a vendor's perspective, if we can't verify the behavior change in `CUDA`, I would prefer using `#ifdef GGML_USE_MUSA` to restrict the code changes to `MUSA` only.

Reply:
Yes, it should be `cross_entropy_loss_f32`.

Reply:
OK, thank for clarifications :D